apps/cowswap-frontend/src/modules/permit/const.ts

```diff
@@ -31,3 +32,10 @@ export const ORDER_TYPE_SUPPORTS_PERMIT: Record<TradeType, boolean> = {
 }
 
 export const PENDING_ORDER_PERMIT_CHECK_INTERVAL = ms`1min`
+
+// DAI's mainnet contract (https://etherscan.io/address/0x6b175474e89094c44da98b954eedeac495271d0f#readContract) returns
+// `1` for the version, while when calling the contract method returns `2`.
```

Comment:
Sorry, I didn't get the problem. I see DAI contract returns version=1, where do we get version=2?

Reply:
```
fetch("https://rpc.ankr.com/eth", {
  "headers": {
    "accept": "*/*",
    "content-type": "application/json",
  },
  "referrer": "http://localhost:3000/",
  "referrerPolicy": "strict-origin-when-cross-origin",
  "body": "{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x6b175474e89094c44da98b954eedeac495271d0f\",\"data\":\"0x54fd4d50\"},\"latest\"],\"id\":49,\"jsonrpc\":\"2.0\"}",
  "method": "POST",
  "mode": "cors",
  "credentials": "omit"
}).then(res => res.json()).then(console.log)
```

It's very odd. I really see that version=2 in the JSON rpc response